svc-lighthouse-api/src/main/java/gov/va/vro/abddataaccess/service/FieldExtractor.java

```diff
@@ -52,6 +55,17 @@ private static String formatDate(Date date) {
     return fmt.format(date);
   }
 
+  public static Coding getAValidCoding(CodeableConcept code) {
+    if (code.hasCoding()) {
+      for (Coding coding : code.getCoding()) {
+        if (coding.hasCode() && !coding.getSystem().trim().startsWith(SNOMED)) {
```

Comment:
Are we no longer filtering out *Missing codes?

Reply:
I thought it didn't need it as "snomed" covered missing code case. But it is better to add it back in case LH FHIR data does it in a different "system". I added it back. Thanks to bring it up.